ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -25,18 +25,23 @@
   when: jobs_env is undefined
 
 - name: create group
-  when: not os|startswith("zos")
+  when: not os|startswith("zos") and not os|startswith("ibmi")
   group: name="{{ server_user }}"
 
 - name: create user
-  when: "not os|startswith('zos')"
+  when: "not os|startswith('zos') and not os|startswith('ibmi')"
   user:
     name: "{{ server_user }}"
     group: "{{ server_user }}"
     shell: "{{ bash_path[os|stripversion]|default('/bin/bash') }}"
 
+- name: create user for ibmi
+  when: os|startswith("ibmi")
+  command: "/QOpenSys/usr/bin/system 'CRTUSRPRF USRPRF({{ server_user }}) PASSWORD(*none)'"
```

Comment:
TASK [jenkins-worker : create user for ibmi] **************************************************
fatal: [test-iinthecloud-ibmi72-ppc64_be-1]: FAILED! => {"changed": true, "cmd": ["/QOpenSys/usr/bin/system", "CRTUSRPRF USRPRF(iojs) PASSWORD(*none)"], "delta": "0:00:00.117563", "end": "202
0-05-12 18:29:21.000635", "msg": "non-zero return code", "rc": 255, "start": "2020-05-12 18:29:20.883072", "stderr": "CPF2214: User profile IOJS already exists.", "stderr_lines": ["CPF2214: U
ser profile IOJS already exists."], "stdout": "", "stdout_lines": []}

@ThePrez

Reply:
Not sure why `ignore_errors: true` doesn't seem to do anything, but a `creates:` would be more elegant anyway. Will make that change.

Reply:
Committed in https://github.com/nodejs/build/pull/1923/commits/a23e8188728f9f31c070fb32aa35d18d7a22a9de